mne/utils.py

```diff
@@ -2015,7 +2016,10 @@ def _get_stim_channel(stim_channel, info, raise_error=True):
     if ch_count > 0:
         return stim_channel
 
-    if 'STI 014' in info['ch_names']:
+    if 'STI101' in info['ch_names']:  # newer Elekta systems
+        return ['STI101']
```

Comment:
Is it so that the stim channel name changed at the same time as the space was removed? Or should we make calls to ``utils._clean_names`` to be sure.